### ISSUE TITLE
Missing redirect for publishing tutorial

### ISSUE THREAD

**Issue by kojoidrissa** [pypa/packaging.python.org#610]
Links in the CPython docs for distributing modules, found [here](https://docs.python.org/3.6/distributing/index.html#reading-the-guide) link to https://packaging.python.org/tutorials/distributing-packages, but these docs have moved to [Packaging Python Projects](https://packaging.python.org/tutorials/packaging-projects/)

I found this with @ncoghlan, and he's working on cleaning it up as I type this.

**Comment by ncoghlan** [comment]
Fixed directly on readthedocs.org - the path with the trailing slash was redirected, but the CPython links omit the slash.

I also changed the `/distributing` redirect to go directly to the new location.

@theacodes I remember we were talking a while back about potentially setting up a version controlled file defining the redirects, but am I right in thinking we never actually set that up, and we're still working directly with entries in ReadTheDocs?
